src/main/java/org/gitlab4j/api/ProtectedBranchesApi.java

```diff
@@ -185,14 +186,15 @@ public ProtectedBranch protectBranch(Object projectIdOrPath, String branchName,
      */
     public ProtectedBranch protectBranch(Object projectIdOrPath, String branchName,
             Integer allowedToPushUserId, Integer allowedToMergeUserId, Integer allowedToUnprotectUserId,
-            Boolean codeOwnerApprovalRequired) throws GitLabApiException {
+            Boolean codeOwnerApprovalRequired, Boolean allowForcedPush) throws GitLabApiException {
```

Comment:
Usually when we add a new parameter we also add a method with the old signature (using java method overloading) to preserve backward compatibility.